### DISCUSSION TITLE
having the completions with cargo make and zsh

### DISCUSSION THREAD

**Discussion by woss** [sagiegurari/cargo-make#578] [Ideas]
I started to move my scripts to the `Makefile.toml` and it seems to me that I will have many tasks. I would like my coworkers don't have to search and find the task names instead of having them as completions. The current sourcing only works with `makers` and on `bash`, we are using zsh and to be specific https://github.com/romkatv/zsh4humans

It is an incredibly good setup with awesome completions which you can find [here](https://github.com/zsh-users/zsh-completions/blob/master/zsh-completions-howto.org) .

I am not any good in zsh programming and i believe that the `cargo make ...completion` would be a great addition and excellent UX with this project.

**Comment by sagiegurari** [comment]
@woss completely missed this one. i get the discussions overlooked. can you please create an issue and i'll check it out?

**Reply by woss** [reply]
https://github.com/sagiegurari/cargo-make/issues/590
